src/main/java/org/codehaus/mojo/exec/AbstractExecMojo.java

```diff
@@ -134,6 +154,19 @@ public abstract class AbstractExecMojo extends AbstractMojo {
     @Parameter(property = "addOutputToClasspath", defaultValue = "true")
     private boolean addOutputToClasspath;
 
+    /**
+     * Indicates if this plugin's dependencies should be used when executing the main class.
+     * <p>
+     * This is useful when project dependencies are not appropriate. Using only the plugin dependencies can be
+     * particularly useful when the project is not a java project. For example a mvn project using the csharp plugins
+     * only expects to see dotnet libraries as dependencies.
+     * </p>
+     *
+     * @since 1.1-beta-1
+     */
+    @Parameter(property = "exec.includePluginsDependencies", defaultValue = "false")
+    protected boolean includePluginDependencies;
```

Comment:
As this property now will be inherited by exec and execJava mojo `since` tag is not accurate true for both ... 
I don't know a good idea for `since` tag ....

Reply:
what is your suggestion?

Reply:
ok, we can put on it ... `since 3.4.0` it will be true for both goals 😄